app/views/widgets/image/_form.slim

```diff
@@ -1,3 +1,4 @@
-=> label_tag 'widgets[image][image_url]', 'Image URL:'
-= text_field_tag 'widgets[image][image_url]'
-= hidden_field_tag 'widgets[image][widget_type]', widget_type_id
+#image_widget_form.form-group
+  => label_tag 'widgets[image][image_url]', 'Image URL:'
+  = text_field_tag 'widgets[image][image_url]', options['image_url']
+  = hidden_field_tag 'widgets[image][widget_type]', WidgetType.find_by(:widget_name => 'image').id
```

Comment:
Yes, this will need to change and I wanted to point this out to you, but widget_type_id is available inside the options only on edit or create - I don't recall which.